src/__experimental__/components/radio-button/radio-button-group.component.js

```diff
@@ -33,22 +76,41 @@ const RadioButtonGroup = (props) => {
       {
         checked,
         inputName: groupName,
-        key,
         onChange: handleChange,
-        tabindex
+        tabindex,
+        hasError,
+        hasWarning,
+        hasInfo
       }
     );
   });
 
+  const icon = () => {
+    const type = getValidationType(props);
+    const message = getValidationMessage(type, props);
+
+    return (
+      <ValidationIconStyle type={ type }>
+        <Icon
+          type={ type }
+          tooltipMessage={ message }
+        />
+      </ValidationIconStyle>
```

Comment:
I think that reusing the `ValidationIcon` component would be a better approach. We could move `InputPresentation` context from that component to the `InputIconToggle` and then everything should be fine.

Reply:
This has been resolved in FE-1728 PR. Waiting for it to be merged.

Reply:
ValidationIcon has been moved into the label component in PR 2278. Those changes have been reflected in this PR.